addons/project/static/src/scss/project_form.scss

```diff
@@ -34,3 +34,8 @@
         color: $o-main-text-color;
     }
 }
+
+.o_form_view .o_group .o_field_widget.o_text_overflow {
+    width: auto !important;
+    min-width: auto;
+}
```

Comment:
This was done to avoid getting a line break between the label "Create tasks by sending an email to" and the actual email address entered in the project edit form view. But maybe there is a better way to get the same result?